### ISSUE TITLE
xdotool bug? It mostly {fails: skips} output of {Unicode characters as curly-quotes} in Manjaro KDE but not in Ubuntu VM

### ISSUE THREAD

**Issue by jameshofstra** [jordansissel/xdotool#185]
Please refer to my detailed post at https://forum.manjaro.org/t/29475

**Comment by jameshofstra** [comment]
Update: I've found an alternate solution using `xmodmap` instead of {`xbindkeys` and `xdotool`}, as noted [here](https://unix.stackexchange.com/questions/387116/#comment689330_387116).

**Comment by FascinatedBox** [comment]
@jameshofstra Do you have more information? Both of your links fail for me.

**Comment by jameshofstra** [comment]
Wow, it's been almost 4 years since I opened this issue. I'm surprised it's gotten some attention now.

Unfortunately, I don't remember any details of the alternate solution I came up with back then, nor do I remember archiving either of my cross posts while they were still visible. And given it's been 4 years, I don't know if the issue still exists in current Manjaro.

I have since moved on - not only from this tool, but also from Manjaro and GitHub. (And from the _very_ weird writing style used in the issue title and the dead cross posts - that's not how I write at all. But I digress.) As such, I will be closing this issue now.
